lib/linguist/languages.yml

```diff
@@ -3289,6 +3289,21 @@ Jupyter Notebook:
   aliases:
   - IPython Notebook
   language_id: 185
+Justfile:
+  type: programming
+  aliases:
+  - Justfile
```

Comment:
An alias isn't needed when the alias is the same as the language name.

```suggestion
```

Reply:
The language should be `just`, since that's the name of the tool/compiler.